SPEC.md

```diff
@@ -847,21 +847,77 @@ Any `${identifier}` inside of a string literal must be replaced with the value o
 
 ```
 $runtime = 'runtime' $ws* '{' ($ws* $runtime_kv $ws*)* '}'
-$runtime_kv = $identifier $ws* '=' $ws* $string
+$runtime_kv = $identifier $ws* '=' $ws* $expression
 ```
 
-The runtime section defines key/value pairs for runtime information needed for this task.  Values are only interpreted by execution backends and will be stored as raw strings until that point. Individual backends will define which keys they will inspect so a key/value pair may or may not actually be honored depending on how the task is run.
+The runtime section defines key/value pairs for runtime information needed for this task.  Individual backends will define which keys they will inspect so a key/value pair may or may not actually be honored depending on how the task is run.
 
-While the key/value pairs are arbitrary, but a few of them have special meaning:
+Values can be any expression and it is up to the engine to reject keys and/or values that do not make sense in that context.  For example, consider the following WDL:
+
+```wdl
+task test {
+  command {
+    python script.py
+  }
+  runtime {
+    docker: ["ubuntu:latest", "broadinstitute/scala-baseimage"]
+  }
+}
+```
+
+The value for the `docker` runtime attribute in this case is an array of values.  The parser should accept this.  Some engines might interpret it as an "either this image or that image" or could reject it outright.
```

Comment:
I'm merging this as-is but for the future IMO it should be clear that engines also have the freedom to ignore keys/values/etc as they see fit, not just reject